packages/Modal/stories/Modal.stories.js

```diff
@@ -106,7 +107,47 @@ storiesOf(`${storyName}/Examples`, module)
         <p>autofocus disabled</p>
       </Modal.Content>
     </Modal>
-  ));
+  ))
+  .add("with focus on heading", () => {
+    const refHeading = React.useRef(null);
+    const [isOpen, setOpen] = React.useState(false);
+
+    return (
+      <Story>
+        <Button
+          onClick={() => {
+            setOpen(true);
+          }}
+        >
+          Open Modal
+        </Button>
+        <Modal
+          isOpen={isOpen}
+          onClose={() => {
+            setOpen(false);
+          }}
+          onAfterOpen={() => {
+            if (refHeading.current) refHeading.current.focus();
+          }}
+          css={`
+            [data-pka-anchor="heading"]:focus {
+              ${stylers.focusRing.subtle()}
+            }
+          `}
```

Comment:
Dotted focus ring just for the storybook examples.

Reply:
can we use dashed intead of dotted? dashed look better IMHO :P 🙈

Reply:
This is a design decision, I don't really care. 
I'll change the `styler` if @sunminsarahlee gives it the 👍